kythe/go/extractors/cmd/bazel/bazel_go_extractor/bazel_go_extractor.go

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 The Kythe Authors. All rights reserved.
+ * Copyright 2020 The Kythe Authors. All rights reserved.
```

Comment:
FYI modifying an existing file generally doesn't invalidate the copyright date. The date here is when the copy right _begins_, so moving it later is arguably misleading—it still applies.

Not a big deal, just something I've seen a few times in passing.

Reply:
The arc linter complained about this file and seemed to indicate the top of the file. After changing the copyright date, I realized I just needed to run gofmt. Changed back to 2016.